### ISSUE TITLE
Error Upgrade to 5.5.1

### ISSUE THREAD

**Issue by monsieurlatte** [hjdhjd/homebridge-unifi-protect#748]
**Describe The Problem:**
Upgrading from 5.5.0 to 5.5.1 errors out

**To Reproduce:**
Choose upgrade to 5.5.1

**Logs:**
USER: root
DIR: /homebridge
CMD: npm install --save homebridge-unifi-protect@latest

npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm ERR! code 1
npm ERR! path /homebridge/node_modules/homebridge-unifi-protect/node_modules/ffmpeg-for-homebridge
npm ERR! command failed
npm ERR! command sh -c node install.js
npm ERR! Downloading ffmpeg from https://github.com/homebridge/ffmpeg-for-homebridge/releases/latest/download/ffmpeg-debian-x86_64.tar.gz
npm ERR! node:events:505
npm ERR!       throw er; // Unhandled 'error' event
npm ERR!       ^
npm ERR! 
npm ERR! Error: EACCES: permission denied, open '/root/.npm/_ffmpeg_for_homebridge/.download'
npm ERR! Emitted 'error' event on WriteStream instance at:
npm ERR!     at emitErrorNT (node:internal/streams/destroy:157:8)
npm ERR!     at emitErrorCloseNT (node:internal/streams/destroy:122:3)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:83:21) {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'open',
npm ERR!   path: '/root/.npm/_ffmpeg_for_homebridge/.download'
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-05-26T20_37_25_307Z-debug-0.log

Command failed. Please review log for details.

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** 1,4,1
* **Node Version:** 16.15.0
* **Homebridge-UniFi-Protect Plugin Version:** 5.5.0
* **UniFi Protect Version:** 2.0
* **UniFi Protect Controller Type:** UDMP
* **UniFi Protect Controller Firmware Version:** 7.1.1
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**doesn't apply
* **Operating System and OS Version:** docker Ubuntu Bionic Beaver (18.04.5 LTS)

**Comment by hjdhjd** [comment]
The issue is related to ffmpeg-for-homebridge not HBUP. 😀 Not that it helps when things don't work...uninstall and reinstall the plugin and things should work. Please inquire in the Discord if you have questions related to this topic. 

**The "fix" such that it is is to simply uninstall and reinstall the plugin - you may have to do this multiple times, but it should ultimately work. Not ideal - we are working on solving the issue in ffmpeg-for-homebridge.**

One more thing: I am keeping this issue locked - please do not post comments like "same issue" - they unnecessarily spam things. Please use the thumbs up option to indicate that. "Same here" comments will be deleted - nothing personal, it's going to be a long few days and I want to ensure we keep a well managed issues list. Thanks.

**Comment by hjdhjd** [comment]
Give v5.5.2 a shot. :smile:

**Comment by hjdhjd** [comment]
Fixed in v5.5.2 with a reversion in ffmpeg-for-homebridge.
